### ISSUE TITLE
NVDA Screen Reader Cannot Announce Selected AI Model in Copilot Chat Panel After Recent Update

### ISSUE THREAD

**Issue by gawachansoft** [microsoft/vscode-copilot-release#9970]
Type: <b>Bug</b>

I am totally blind and use a screen reader, NVDA.  
In the GitHub Copilot chat panel, I am unable to check some information.  
When I press Ctrl+Alt+., the list of AI models is displayed, but I cannot know which AI model is currently selected through NVDA's speech output.  
Until the update in early May, I was able to confirm which AI model was checked using NVDA's speech.  
Please fix this issue.  
I am using the latest versions of both VSCode and GitHub Copilot.  
My NVDA version is 2024.4.2JP.

Extension version: 0.27.1
VS Code version: Code 1.100.2 (848b80aeb52026648a8ff9f7c45a9b0a80641e2e, 2025-05-14T21:47:40.416Z)
OS version: Windows_NT x64 10.0.26100
Modes:

<details><summary>Logs</summary>
<pre>
Trace: Resolving chat model
Trace: Resolving chat model
Trace: Resolved chat model
Trace: Resolved chat model
Trace: Processed intent
Trace: Resolving chat model
Trace: Resolved chat model
Trace: Built prompt
Trace: Sending prompt to model
Info: message 0 returned. finish reason: [stop]
Info: request done: requestId: [b6fbcf48-73cc-45a6-9f97-d163468e2c83] model deployment ID: []
Trace: Resolving chat model
Trace: Resolved chat model
Info: message 0 returned. finish reason: [stop]
Info: request done: requestId: [2532a781-6186-4b66-8b88-e347d7c980fd] model deployment ID: []
Trace: Resolving chat model
Trace: Resolved chat model
Trace: Building intent detector
Trace: Resolving chat model
Trace: Resolved chat model
Trace: Built intent detector
Info: message 0 returned. finish reason: [stop]
Info: request done: requestId: [d945cae2-b4ac-4476-b921-c40047d08b71] model deployment ID: []
Trace: Resolving chat model
Trace: Resolved chat model
Trace: Resolving chat model
Trace: Resolved chat model
Trace: Resolving chat model
Trace: Resolved chat model
Trace: [panel] chat request received from extension host
Trace: Processing intent
Trace: Resolving chat model
Trace: Resolving chat model
Trace: Resolved chat model
Trace: Resolved chat model
Trace: Processed intent
Trace: Resolving chat model
Trace: Resolved chat model
Trace: Built prompt
Trace: Sending prompt to model
Info: message 0 returned. finish reason: [stop]
Info: request done: requestId: [c6c12f30-70cb-4d6d-8fa0-8ab45b6c5228] model deployment ID: []
Debug: [context keys] Window state change. Needs offline check: false, active: true, focused: false.
Debug: [context keys] Window state change. Needs offline check: false, active: true, focused: true.
Debug: [context keys] Window state change. Needs offline check: false, active: true, focused: false.
Debug: [context keys] Window state change. Needs offline check: false, active: false, focused: false.
Debug: [context keys] Window state change. Needs offline check: false, active: false, focused: true.
Debug: [context keys] Window state change. Needs offline check: false, active: true, focused: true.
Debug: [context keys] Window state change. Needs offline check: false, active: true, focused: false.
Debug: [context keys] Window state change. Needs offline check: false, active: true, focused: true.
</pre>
</details><details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Celeron(R) N4000 CPU @ 1.10GHz (2 x 1094)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: disabled_off<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|3.83GB (0.55GB free)|
|Process Argv|--crash-reporter-id cd01e917-9080-49e4-9d5c-d6e1976e2a14|
|Screen Reader|yes|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dwnewjupyter:31046869
pythonrstrctxt:31112756
nativeloc2:31192216
5fd0e150:31155592
dwcopilot:31170013
6074i472:31201624
dwoutputs:31242946
customenabled:31248079
9064b325:31222308
copilot_t_ci:31222730
e5gg6876:31282496
pythoneinst12:31285622
bgtreat:31268568
4gafe986:31271826
c7cif404:31309971
996jf627:31283433
pythonrdcb7cf:31303019
usemplatestapi:31297334
0aa6g176:31307128
7bj51361:31289155
747dc170:31275177
6518g693:31309498
pylancecolorcf:31309109
aj953862:31281341
generatesymbolt:31295002
convertfstringf:31295003
gendocf:31295004

```

</details>
